packages/docs/versioned_docs/version-2.0.0-alpha-3/protocol/04-reputation.md

```diff
@@ -11,6 +11,7 @@ Attesters define the reputation system for their application on top of the UniRe
   * `posRep` is the positive reputation given by the attester
   * `negRep` is the negative reputation given by the attester
   * `graffiti` is the message given by the attester
+    * TODO: give examples of graffiti usage
```

Comment:
I am still unclear on the purpose of graffiti. Is this something the attester chooses how to use for their own application?

Reply:
Yes, attesters can choose how they use this data. For example a user can be given a username as graffiti.